src/parser/jobs/brd/modules/Util.tsx

```diff
@@ -0,0 +1,155 @@
+/**
+ * @author Yumiya
+ * @author Ririan
+ */
+
+import {DamageEvent, Event} from 'fflogs'
+import Module, {dependency} from 'parser/core/Module'
+import Combatants from 'parser/core/modules/Combatants'
+import Downtime from 'parser/core/modules/Downtime'
+import Enemies from 'parser/core/modules/Enemies'
+import Invuln from 'parser/core/modules/Invulnerability'
+import Timeline from 'parser/core/modules/Timeline'
+import React from 'react'
+import {Button} from 'semantic-ui-react'
+import AdditionalStats from './AdditionalStats'
+
+const CONVERSION_FACTOR = 0.1
+
+const DHIT_MOD = 1.25
+
+const TRAIT_STRENGTH = 0.20
+
+interface StatusEffect {
+	id: number
+	name: string
+	icon: string
+	duration?: number
+	amount?: number
+}
+interface DowntimeInfo {
+	start: number
+	end: number
+}
+interface AdditionalDamageEvent extends DamageEvent {
+	criticalHit: number
+	directHit: number
+}
+
+export default class Util extends Module {
+	static handle = 'util'
+
+	@dependency private combatants!: Combatants
+	@dependency private downtime!: Downtime
+	@dependency private enemies!: Enemies
+	@dependency private invuln!: Invuln
+	@dependency private timeline!: Timeline
+	@dependency private additionalStats!: AdditionalStats
+
+	hasBuff(status: StatusEffect) {
```

Comment:
If there are returning stuff, probably a good idea to type the returns.
```suggestion
	hasBuff(status: StatusEffect): boolean {
```